tests/suites/test_suite_pem.function

```diff
@@ -32,6 +32,35 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE depends_on:MBEDTLS_PEM_WRITE_C */
+void mbedtls_pem_write_buffer_lengths()
+{
+    unsigned char data[256] = {};
```

Comment:
Does anyone know what the cross-platform way to do this is?

Reply:
I tried locally with 
```suggestion
    unsigned char data[256] = { 0 };
```
and it seemed to pass that test.

Reply:
In Mbed TLS, for an array: `{ 0 }` (`{}` is not in C99). For a struct: you have to write all the fields, otherwise clang complains (at a warning level we care about, which is mostly `-Wall -Wextra`).